.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,30 +1,34 @@
 ---
 name: Bug report
-about: Create a bug report to help us improve the VSTS plugin
-
+about: Create a bug report to help us improve the AWS Toolkit for Azure Devops
```

Comment:
It's "Azure DevOps": https://azure.microsoft.com/en-us/services/devops/

Reply:
I'm not sure why I took a task that requires good spelling, fixed.